src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,60 @@
+package frc.trigon.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+
+public class ArmConstants {
+    private static final double
+            MAX_ANGLE_VELOCITY = 150,
+            MAX_ANGLE_ACCELERATION = 190,
+            MAX_ELEVATOR_VELOCITY = 2,
+            MAX_ELEVATOR_ACCELERATION = 3;
+    static final TrapezoidProfile.Constraints
+            ANGLE_CONSTRAINTS = new TrapezoidProfile.Constraints(
+            MAX_ANGLE_VELOCITY, MAX_ANGLE_ACCELERATION
+    ),
+            ELEVATOR_CONSTRAINTS = new TrapezoidProfile.Constraints(
+                    MAX_ELEVATOR_VELOCITY, MAX_ELEVATOR_ACCELERATION
+            );
+
+    private static final double
+            ARM_MECHANISM_WIDTH = 4,
+            ARM_MECHANISM_HEIGHT = 4;
+    private static final double
+            ARM_ROOT_WIDTH = 1,
+            ARM_ROOT_LENGTH = 1;
+    private static final double
+            ARM_LIGAMENT_LENGTH = 2,
+            ARM_LIGAMENT_ANGLE = 2;
+    private static final double ARM_MECHANISM_LENGTH = 1;
+    static final Mechanism2d ARM_MECHANISM = new Mechanism2d(ARM_MECHANISM_WIDTH, ARM_MECHANISM_HEIGHT);
+    private static final MechanismRoot2d
+            ARM_ROOT = ARM_MECHANISM.getRoot("ArmRoot", ARM_ROOT_WIDTH, ARM_ROOT_LENGTH),
+            TARGET_POSITION_ROOT = ARM_MECHANISM.getRoot("TargetPositionRoot", ARM_ROOT_WIDTH, ARM_ROOT_LENGTH);
+    static final MechanismLigament2d
+            ARM_LIGAMENT = ARM_ROOT.append(new MechanismLigament2d("ArmLigament", ARM_LIGAMENT_LENGTH, ARM_LIGAMENT_ANGLE, ARM_MECHANISM_LENGTH, new Color8Bit(Color.kBlue))),
+            TARGET_POSITION_LIGAMENT = TARGET_POSITION_ROOT.append(new MechanismLigament2d("TargetPositionLigament", ARM_LIGAMENT_LENGTH, ARM_LIGAMENT_ANGLE, ARM_MECHANISM_LENGTH, new Color8Bit(Color.kGray)));
```

Comment:
The `ARM_LIGAMENT_ANGLE` should just be 0, and the fourth param you input should be the line width. For good visualization, this should be 10. Have a constant for it.